tests/google_service_test.py

```diff
@@ -0,0 +1,59 @@
+import os
+import requests
+from base_app.base import app
+from base_app.google_service import GoogleService
+from dotenv import load_dotenv
+load_dotenv()
+
+def test_can_initialize():
+    service = GoogleService()
+    assert isinstance(service, GoogleService)
+
+def test_google_api_returns_successful_response():
```

Comment:
@linda-le1 – in addition to the params, do these tests not also require a URL which to connect?